apps/explorer/src/pages/address-result/AddressResult.tsx

```diff
@@ -4,10 +4,10 @@
 import { useParams } from 'react-router-dom';
 
 import { ErrorBoundary } from '../../components/error-boundary/ErrorBoundary';
-import { TransactionsForAddress } from '../../components/transactions/TransactionsForAddress';
 
 import OwnedCoins from '~/components/OwnedCoins/OwnedCoins';
 import OwnedObjects from '~/components/OwnedObjectsV2/OwnedObjects';
+import TransactionBlocks from '~/components/TransactionBlocks/TransactionBlocks';
```

Comment:
Not totally sure about the naming, let me know if this could be something else!

Reply:
I think `TransactionBlocksForAddress` makes the most sense.